test/controllers/lookups_controller_test.rb

```diff
@@ -78,7 +80,10 @@ def test_lookup_name
     assert_redirected_to(name_path(names(:agaricus_campestris).id))
 
     get(:lookup_name, params: { id: "Agaricus newname" })
-    assert_redirected_to(names_path)
+    assert_redirected_to(
+      glossary_terms_path,
+      "It should go to Glossary index when there's no Name or Glossary match"
+    )
```

Comment:
Agreed.  Glossary index is probably cheaper(?) and it will have the effect of "advertising" this new feature whenever someone clicks on a link that doesn't match either.